lmfdb/local_fields/main.py

```diff
@@ -304,6 +304,18 @@ def galcolresponse(n,t,cache):
         return 'not computed'
     return group_pretty_and_nTj(n, t, cache=cache)
 
+def formatbracketcol(blist):
+    if blist == []:
+        return r'$[\ ]$'
+    if not blist:
+        return 'not computed'
+    return f'${blist}$'
+
+def intcol(j):
+    if not j:
```

Comment:
This should be `if j is None`, since this will display `not computed` for `j=0`.  I know that's not a possible value for `t` and `u`, but this code could eventually get copied and used elsewhere.

I'll go ahead and make this change.